README.md

```diff
@@ -182,8 +183,9 @@ class App extends React.Component {
 
 #### Event handling
 
+- `onMount`: required. `function(initialValues:object)` function that gets called immediately after form is mounted. `initialValues` is an object contains all form fields and their corresponding default value.
```

Comment:
wait, why is this event handler required instead of optional? Not everyone will need this. Also, wouldn't everyone already know what all the initial values are, because any field with a defaultValue will have that value, and any field without a defaultValue will be falsey? Why is onMount necessary? (what use-case does it address?)

Reply:
Ah I just thought it would be easier for formbuilder user to get all values at once instead of manually copy `defaultValue` one by one from the `field` object

Reply:
Why would they have to do that at all, though? We already have a mechanism to communicate value changes through `onChange`, so if there is a default value, then we can trigger `onChange` for that initial value change, and the code will automatically notify the parent of the value update.